js/core/devices.js

```diff
@@ -24,10 +24,8 @@ var KNOWN_UA_TABLE = {
     "androidPhone": "Android Mobile",
     "androidTablet": "Android",
     "win8": "MSAppHost",
```

Comment:
How about `win8` and `win10`? They look redundant here because you removed `win` from uaParsers.